modules/mono/godotsharp_dirs.cpp

```diff
@@ -179,16 +179,16 @@ class _GodotSharpDirs {
 
 #ifdef OSX_ENABLED
 		if (!DirAccess::exists(data_editor_tools_dir)) {
-			data_editor_tools_dir = exe_dir.plus_file("../Frameworks/GodotSharp/Tools");
+			data_editor_tools_dir = exe_dir.plus_file("../Resources/GodotSharp/Tools");
 		}
 
 		if (!DirAccess::exists(data_editor_prebuilt_api_dir)) {
-			data_editor_prebuilt_api_dir = exe_dir.plus_file("../Frameworks/GodotSharp/Api");
+			data_editor_prebuilt_api_dir = exe_dir.plus_file("../Resources/GodotSharp/Api");
 		}
 
 		if (!DirAccess::exists(data_mono_root_dir)) {
 			data_mono_etc_dir = exe_dir.plus_file("../Resources/GodotSharp/Mono/etc");
-			data_mono_lib_dir = exe_dir.plus_file("../Frameworks/GodotSharp/Mono/lib");
+			data_mono_lib_dir = exe_dir.plus_file("../Resources/GodotSharp/Mono/lib");
```

Comment:
Do we need a new dir string to keep track of the fact that OSX has some lib stuff in `../Resources/GodotSharp/Mono/lib` and other in `../Frameworks/GodotSharp/Mono/lib`? Not also that the `!TOOLS_ENABLED` codepath uses `Frameworks` while this one uses `Resources`.

Reply:
This is likely the reason it cannot find the Mono native libs. We append this directory to PATH for Mono to search for the libs there: https://github.com/godotengine/godot/blob/43264c63724ef040606936c65a1c24edb9d1f862/modules/mono/mono_gd/gd_mono.cpp#L240-L248

Reply:
Resolved since we put all in `Resources`.